helm-app/templates/users-service-db.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Values.usersServiceDb.statefulsetName }}
+  namespace: {{ .Values.namespace }}
+spec:
+  serviceName: "{{ .Values.usersServiceDb.service.name }}"
+  replicas: {{ .Values.usersServiceDb.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Values.usersServiceDb.selector.app }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.usersServiceDb.selector.app }}

```

Comment:
Any particular reason I am not able to add any more labels?

Reply:
Fixed.

Reply:
Fixed for more labels.